lua/neogit/status.lua

```diff
@@ -1238,6 +1238,10 @@ function M.create(kind, cwd)
   }
 end
 
+function M.is_dirty()
```

Comment:
This should go to `lib/git/status.lua` - this file is for status _buffer_ logic :)